test/e2e/flowaggregator_test.go

```diff
@@ -0,0 +1,334 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+/* Sample output from the collector:
+IPFIX-HDR:
+  version: 10,  Message Length: 288
+  Exported Time: 1605749238 (2020-11-19 01:27:18 +0000 UTC)
+  Sequence No.: 9,  Observation Domain ID: 2134708971
+DATA SET:
+  DATA RECORD-0:
+    flowStartSeconds: 1605749227
+    flowEndSeconds: 2288912640
+    sourceIPv4Address: 10.10.0.27
+    destinationIPv4Address: 10.10.0.28
+    sourceTransportPort: 34540
+    destinationTransportPort: 5201
+    protocolIdentifier: 6
+    packetTotalCount: 1037047
+    octetTotalCount: 45371902943
+    packetDeltaCount: 410256
+	octetDeltaCount: 18018632100
+	originalExporterIPv4Address: 10.10.0.1
+	originalObservationDomainId: 2134708971
+    reversePacketTotalCount: 854967
+    reverseOctetTotalCount: 44461736
+    reversePacketDeltaCount: 330362
+    reverseOctetDeltaCount: 17180264
+    sourcePodName: perftest-a
+    sourcePodNamespace: antrea-test
+    sourceNodeName: k8s-node-master
+    destinationPodName: perftest-b
+    destinationPodNamespace: antrea-test
+    destinationNodeName: k8s-node-master
+    destinationClusterIPv4: 10.103.234.179
+    destinationServicePortName: antrea-test/perftest-b:
+    ingressNetworkPolicyName: test-networkpolicy-ingress
+    ingressNetworkPolicyNamespace: antrea-test
+    egressNetworkPolicyName: test-networkpolicy-egress
+    egressNetworkPolicyNamespace: antrea-test
+
+Intra-Node: Flow record information is complete for source and destination e.g. sourcePodName, destinationPodName
+Inter-Node: Flow record from destination Node is ignored, so only flow record from the source Node has its K8s info e.g., sourcePodName, sourcePodNamespace, sourceNodeName etc.
+AntreaProxy enabled (Intra-Node): Flow record information is complete for source and destination along with K8s service info such as destinationClusterIP, destinationServicePort, destinationServicePortName etc.
+AntreaProxy enabled (Inter-Node): Flow record from destination Node is ignored, so only flow record from the source Node has its K8s info like in Inter-Node case along with K8s Service info such as destinationClusterIP, destinationServicePort, destinationServicePortName etc.
+*/
+
+func TestFlowAggregator(t *testing.T) {
```

Comment:
I think TestFlowAggregator and TestFlowExporter can be consolidated as in both cases, we need an external flow collector and just the checks will be different. This will save some time in deploying and tearing down the external flow collector.